module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -945,11 +947,11 @@ protected function getVideo(
     /**
      * Function to return document in associative array
      *
-     * @param string $url           Url of the document
-     * @param string $format        Format of the document
-     * @param string $description   Description of the document
-     * @param array  $rights        Array of document rights
-     * @param bool   $displayAsLink Display the document as a link, default is false
+     * @param string $url         Url of the document
+     * @param string $format      Format of the document
+     * @param string $description Description of the document
+     * @param array  $rights      Array of document rights
+     * @param bool   $linkType    Type of link document is displayed as, default is 'proxy'
```

Comment:
Kommentti vähän sekava, laita vaikka "Type of document link, default is 'proxy-link'".